### ISSUE TITLE
Building Tensorflow & Segmentation Acc.

### ISSUE THREAD

**Issue by migari01** [microsoft/O-CNN#124]
Hello,

I was trying to build/run your code and faced some minor issues. 

1.  Building the tensorflow code with script build.py was failing because it could not find the "object" directory. This can be solved by adding an else statement [here](https://github.com/microsoft/O-CNN/blob/78ef8dec907c19345b0fbbecaf84fbfa141c6193/tensorflow/libs/build.py#L157).
For example:
```
if os.path.exists("object"):
  os.system("rm -r object")
  os.mkdir("object")
else:
  os.mkdir("object")
os.system("make -j all")
```
2. While running run_seg_partnet.py for the bed category, I noticed the test accuracy was always _nan_. I looked through the data and code to see what caused it, and this was happening because all the point labels of the last model in the validation split (Bed_447c50e658d729f044ededc21cf0e35e_11309_level3) were 0. In your code you ignore all points with label zero, thus for this model no points are left to process, which causes the _nan_ value. So, this model can either be removed from the split or handle the 0 label differently, for example ignore it only in the IoU calculations. Also not masking labels with that value solves the issue, but I suppose this is not what you wanted. Do you have any other suggestions how to handle this case?

**Comment by wang-ps** [comment]
1. Thank you so much for figuring out this bug, I have updated the code.
2. I also noticed this issue when doing experiments. The points with the label 0 in PartNet are background points ( i.e., unlabeled points), and they should not be considered in the calculation of IoU. According to the formula used to calculate IoU, the point cloud containing only points with the label 0 does not affect the final value of the mean IoU. However, it will produce a `nan` when using the `tf.reduce_mean` to calculate the accuracy.  You can safely ignore this issue since the accuracy is not used as the evaluation metric. You can also delete the point cloud only containing the label 0 if you find it and do want to get the accuracy.
